tests/utils_tests/test_html.py

```diff
@@ -66,6 +69,37 @@ def test_format_html(self):
             "&lt; Dangerous &gt; <b>safe</b> &lt; dangerous again <i>safe again</i>",
         )
 
+    def test_format_html_join_with_positional_arguments(self):
+        self.assertEqual(
+            format_html_join(
+                "\n", "<li>Id: {}</li><li>Text: {}</li>", ((1, "Text1"), (2, "Text2"))
+            ),
+            "<li>Id: 1</li><li>Text: Text1</li>\n<li>Id: 2</li><li>Text: Text2</li>",
+        )
+
+    def test_format_html_join_with_keyword_arguments(self):
+        Category.objects.create(name="c1")
+        Category.objects.create(name="c2")
+        Category.objects.create(name="c3")
+
+        self.assertEqual(
+            format_html_join(
+                "\n",
+                "<li>Id: {id}</li><li>Text: {text}</li>",
+                ({"id": 1, "text": "Text1"}, {"id": 2, "text": "Text2"}),
```

Comment:
Use a list per [this rule of thumb](https://stackoverflow.com/a/626871/1427135).

```suggestion
                [{"id": 1, "text": "Text1"}, {"id": 2, "text": "Text2"}],
```